lib/resources/r4/slot_patch.yaml

```diff
@@ -0,0 +1,20 @@
+---
+name: Slot
+field_name_base_url: https://hl7.org/fhir/r4/slot-definitions.html#Slot
+operations:
+  - name: replace-status
+    path: /status
+    operation: replace
+    type: code
+    url: https://hl7.org/fhir/R4/slot-definitions.html#Slot.status
+    description: The status of the slot.
+    example: |
+      {
+        "op": "replace",
+        "path": "/status"
+        "status": "free"
+      }
+    note: >
+      <ul>
+      <li>Only `free` and `busy tentative`.</li>
```

Comment:
Maybe we can update the wording to say something like:

'Only `free` and `busy tentative` statuses are accepted for patching.'

Thoughts?

Reply:
nvm. Looks like we already have a conversation about this here: https://github.com/cerner/fhir.cerner.com/pull/608#discussion_r612717160